### PR TITLE
[Fixes] Expose user ID in `userstatus` endpoint

### LINKED ISSUE #281: Expose user ID in `userstatus` endpoint
To help address https://github.com/mozilla/network-pulse/issues/916, we need an easy way to access the user's id. If a user is logged in, we should add another key to `/userstatus` with their ID.

### PR DESCRIPTION
**Pull request by bhavyaagg**
Signed-off-by: Bhavya Aggarwal bhavyaaggarwal1209@gmail.com

Fixes #281 

Updated the pulseapi/users/views.py and templates/users/userstatus.json files.

### CODE REVIEW

**Comment by bhavyaagg** [comment]
@alanmoo 
Please review whenever possible. 
Also, should i use user.id or user.profile.id. Both are giving the same value when i checked on my local server.
Thank you.

**Comment by alanmoo** [comment]
You should use `profile`. Users were created before profiles, but profiles can exist without users (I believe when you put in a creator that isn't an existing user, a "profile" is created)

**Comment by bhavyaagg** [comment]
@alanmoo I will do that straight away.

**Comment by bhavyaagg** [comment]
@alanmoo I have updated the PR with the required update.

**Comment by alanmoo** [comment]
This seems to work! Can you update `users/test_views.py` to test this?

**Comment by bhavyaagg** [comment]
Sure @alanmoo

**Comment by bhavyaagg** [comment]
@alanmoo I have updated the PR.

**Comment by bhavyaagg** [comment]
@alanmoo 
In local environment, the profileid for the user which is created during tests comes out to be '1' but during the Travis build, it is coming to be '198'.

**Comment by alanmoo** [comment]
That's not surprising, other things might be happening in the database.  It doesn't really matter what the number is though, you probably just want to check that it exists and is the right type.

**Comment by bhavyaagg** [comment]
@alanmoo Thank you for your review.
I have re updated my PR.
Thank you

File: pulseapi/users/test_views.py [hunk 1]
```diff
@@ -46,4 +46,6 @@ def test_user_status_true(self):
         json_obj = json.loads(string)
         self.assertEqual(json_obj['loggedin'], True)
         self.assertEqual(json_obj['email'], 'test@example.org')
+        self.assertTrue(json_obj['profileid'])
+        self.assertIsInstance(json_obj['profileid'], type("a"))
```
Comment (alanmoo) [inline pulseapi/users/test_views.py]: I get what this line is doing but it feels odd. Is this some convention I'm not familiar with, or is there a more direct way to check if something is a string in Python? @gideonthomas

Comment (bhavyaagg) [reply pulseapi/users/test_views.py]: I think i have found a better way to do this.

**Comment by bhavyaagg** [comment]
@alanmoo I have updated the PR. Thanks a lot for reviewing so fast. :)

**Comment by alanmoo** [comment]
Looks great @bhavyaagg, thanks!

**Comment by bhavyaagg** [comment]
Thank you @alanmoo

**Pull request merged**